src/pages/hegel/reference/mechanical-object/first-paragraph.mdx

```diff
@@ -1,17 +1,36 @@
+Before even thinking about the `mechanical object`, let us just think about the conceptual structure that presents itself at the beginning of Mechanism. The first determination is described in the following terms: 
 
-First, the mediation  of the syllogism, whatever that means in concrete terms, has been sublated . Strictly speaking, Hegel writes that it has been “balanced out” or “equilibrated” [the German being ausgeglichen]. It is because the mediation of the syllogism has been equilibrated that it was sublated.  As such, the mediation of the syllogism is not nullified but has been set aside by a more developed kind of relation – the relation of the “mechanical object” that is now an immediate identity [unmittelbare Identität]. The moments of the “mechanical object” are immediately identical to each other, and not mediated. 
-The immediacy  of their identity points to the fact that there is no moment of mediation to establish their identity, they just are identical. One way to understand immediacy is to think of how someone might theorise about perception: one might think that what we perceive is the raw sense-data of what is out there and that there is nothing more or less in our perception of the world. What we perceive is immediately identical to what is actually out there. In this sense, there is no moment in-between the moments of the “mechanical object” that conceptually mediate their identity to each other – they simply and immediately are identical to each other. Their identity is, in turn, explained by their immediacy. If one moment is immediately the other moment then there is no distinguishing them from each other. Let’s go back to our example of a philosophical theory of perception that treats perception as an unmediated activity whereby the perceiver receives the raw sense-data as it is in reality.  In this theory, there is an implicit identity between the perceiver and the perceived because if the perceiver does nothing to the sense-date upon perceiving it (such as, say, mediating it through certain concepts) then the perceiver is, in a sense, ontologically the same as the sense-data being perceived. Now, obviously, the moments of the “mechanical object” do not perceive each other but they do relate to each other, and they relate to each other immediately and, as such, they relate to each other as identical moments. It is for these reasons that Hegel begins his account of the “mechanical object” by stating that its moments have “become an immediate identity” (Miller, 711). 
-What exactly are these moments of the “mechanical object” that have become an immediate identity? Hegel clarifies this in the following sentence:
+>“The object is, as we have seen, the syllogism, whose mediation has been sublated and has therefore become an immediate identity” (Hegel, 1991, 711). Let’s unpack this initial thought, without trying to unpack the reference to `the syllogism`.  
```

Comment:
You left some of your own text inside of the block quote. 

![image](https://github.com/user-attachments/assets/98a5e343-4023-4d9b-97bb-2a585e8ca235)

You can also remove the quotation marks when using block quotes.